src/System.Collections/tests/Generic/List/List.Generic.Tests.Remove.cs

```diff
@@ -51,6 +51,12 @@ public void RemoveAll_DefaultElements(int count)
             Assert.Equal(expectedCount, removedCount);
         }
 
+        [Fact]
+        public void RemoveAll_NullMatchPredicate()
+        {
+            Assert.Throws<ArgumentNullException>(() => new List<T>().RemoveAll(null));
```

Comment:
Nit: same comment as on the previous PR, but it'd be good to include parameter name information in Throws for ArgumentExceptions